zoo-css-challenge/index.html

```diff
@@ -78,6 +79,19 @@ <h3>Height and Size</h3>
 			<p>Giraffes use their height to good advantage and browse on leaves and buds in treetops that few other animals can reach (acacias are a favorite). Even the giraffe's tongue is long! The 21-inch tongue helps them pluck tasty morsels from branches. Giraffes eat most of the time and, like cows, regurgitate food and chew it as cud. A giraffe eats hundreds of pounds of leaves each week and must travel miles to find enough food.</p>
 		</section>
 
+		<section class="horse">
+			<h3>Horses</h3>
```

Comment:
```suggestion
			<h2>Horses</h2>
```
All the other animal sections have `h2` as their first child element so it makes sense to have this as `h2` as well so it's at the same heading level

Reply:
Great, Thank you very much